en/docs/references/ballerina-expression-syntax.md

```diff
@@ -1,422 +1,495 @@
 
-#### Examples
+###### Examples:
 
-- Checking if the value of a variable (`firstNumber`) is greater than a literal.
+To read the value of the query parameter `category`, use the following expression:
 
-  ```ballerina
-  firstNumber > 10
-  ```
+```ballerina
+request.getQueryParamValue("category")
+```
   
-- Checking if the value of a variable (`firstNumber`) is less than or equal to the value of another variable (`secondNumber`).
-
-  ```ballerina
-  firstNumber <= secondNumber
-  ```
+To read the value of the query parameter passing the query parameter name as a variable, use the following expression:
 
-## HTTP request related operations
+```ballerina
+request.getQueryParamValue(queryParamName)
+```
 
-All the HTTP-related service applications created in Choreo have a variable called `request` (of type `http:Request`) which the developers can use to query information related to the HTTP request. Following is a list of such common use cases.
+### Read the cookies available in the request
 
-### Reading a header value in the request
-A typical HTTP request contains many headers.  These include standard and custom headers. You can use the `request` variable's `getHeader` function to read a specific header value in any expression-enabled input field.
+HTTP cookies keep stateful information against a client. You can use the `request` variable's `getCookies` method to read cookies present in the request as an array.
 
-#### Examples
+###### Example:
 
-- Reading the value of an HTTP header (`Access-Control-Allow-Origin`) passing the header name.
+To read all the cookies available in the request, use the following expression:
 
-  ```ballerina
-  check request.getHeader("Access-Control-Allow-Origin")
-  
-  ```
-  
-- Reading the value of an HTTP header by passing a variable that contains the header name (`headerName`). Note the omission of the double-quotes.
-
-  ```ballerina
-  check request.getHeader(headerName)
-  ```
+```ballerina
+request.getCookies()
+```
 
-### Checking whether a header is present in the request
+## JSON
 
-Applications sometimes need to check the availability of a header before proceeding to something like reading the header value. You can use the  `hasHeader`  function in the `request` object for this purpose.
+JSON is a textual format for representing a collection of values: a simple value, an array of values, or an object. 
 
-#### Examples
+### Simple values in JSON 
 
-- Checking if a particular HTTP header (`content-length`) exists given the header name.
+Ballerina has a single type named `json` that can represent any JSON value. You can use `json` which is a built-in union type in Ballerina, to store values of the types `nil` (as the null value), `boolean`, `int`, `float`, `decimal`, `string`, `json[]`, and `map<json>`. 
```

Comment:
The comma after "Ballerina" ios not needed.

Reply:
Fixed